src/RaceTimeAverage.js

```diff
@@ -0,0 +1,54 @@
+const calcTotalTime = (endTimeString) => {
+  const endTimeArr = endTimeString.split(/, (?=\d{2})/)
```

Comment:
This is what fixed the bug to allow me to split a string into the array to iterate through.  Using regular expression - that split says split at the ", " (comma followed by a space) when it is immediately followed by 2 digit numbers.